README.md

```diff
@@ -254,6 +254,41 @@ for m in state.messages():
 print(state["answer_1"])
 ```
 
+### OpenLLM
+
+[OpenLLM](https://github.com/bentoml/OpenLLM) allows you to easily run Qwen2.5 as OpenAI-compatible APIs. You can start a model server using `openllm serve`. For example:
+
+```bash
+openllm serve qwen2.5:7b
+```
+
+The server is active at `http://localhost:3000/`, providing OpenAI-compatible APIs. You can create an OpenAI client to call its chat API:
+
```

Comment:
the code in the following seems a bit repetitive to me. I think it's okay to just state that an OpenAI API is availble at ...

Reply:
Deleted. Users can just check out the details in doc.